usaspending_api/settings.py

```diff
@@ -845,3 +845,23 @@
     "original_loan_subsidy_cost": "original_loan_subsidy_cost",
     "business_funds_indicator": "business_funds_indicator"
 }
+
+# If caches added or renamed, edit clear_caches in usaspending_api/etl/helpers.py
+CACHES = {
```

Comment:
I suppose it's clearer just to delete mention of the unused caches.  I stopped in the middle of implementing them...